ui/src/criteria/concept.tsx

```diff
@@ -212,12 +212,17 @@ function ConceptEdit(props: ConceptEditProps) {
           columns={columns}
           disableSelectionOnClick
           className="criteria-concept"
+          disableVirtualization={!!process.env.JEST_WORKER_ID}
```

Comment:
This is not a great code smell; assuming you've already explored better workarounds, I would put a comment here explaining why this is necessary.

Reply:
Added a comment. jsdom, which jest uses for testing, has incomplete layout sizing support so virtualization ends up not displaying most of the columns.